Monika After Story/game/script-affection.rpy

```diff
@@ -946,78 +946,99 @@ init 15 python in mas_affection:
 
         ## DISTRESSED quips
         quips = [
+            _("..."),
             _("...Yes?"),
-            _("...Oh?"),
-            _("...Huh?"),
             _("...Hm?"),
-            _("We can try talking, I guess."),
-            _("I guess we can talk."),
-            _("Oh... You want to talk?"),
-            _("If you want to talk, go ahead."),
-            _("We can talk if you really want to."),
```

Comment:
Not sure why you would want to remove these, I feel they work just fine in that context.
Maybe move them to Upset instead I guess, aside from `"Oh... You want to talk?"` which probably still fits better here.